src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/tests/test_storage.py

```diff
@@ -15,23 +15,25 @@
 from six import StringIO
```

Comment:
Neither pylint or flake8 list it as unused, they are pretty good at figuring out unused imports. So I say leave it out of this PR.